examples/material_gallery/apk/AndroidManifest.xml

```diff
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I ended with these file deletion because I followed this suggestion from flutter start:

```
Warning: Flutter now looks for Android resources in the android/ directory; consider renaming your 'apk/' directory to 'android/'.

Using pre-built SkyShell.apk.
Starting lib/demo/colors_demo.dart on Nexus 5...
Showing logs for Android:
[Android] --------- beginning of main
[Android] --------- beginning of system
[Android] --------- beginning of crash
[Android] F/libc    : Fatal signal 6 (SIGABRT), code -6 in tid 9919 (ui_thread)
[Android] W/ActivityManager:   Force finishing activity 1 org.domokit.sky.shell/.SkyActivity
```

I will try and sort out what's going on here.